extensions/ql-vscode/package.json

```diff
@@ -329,6 +329,36 @@
           "patternErrorMessage": "Please enter a valid GitHub repository",
           "markdownDescription": "[For internal use only] The name of the GitHub repository in which the GitHub Actions workflow is run when using the \"Run Variant Analysis\" command. The repository should be of the form `<owner>/<repo>`)."
         },
+        "codeQL.variantAnalysis.filterResults": {
```

Comment:
Should we call these `defaultFilterResults` and `defaultSortResults` to make it clearer that these options can still be changed?